src/api/app/assets/javascripts/webui/live_build_log.js

```diff
@@ -20,6 +21,12 @@ $.extend(LiveLog.prototype, {
   initialize: function() {
     this.startButton.click($.proxy(this.start, this));
     this.stopButton.click($.proxy(this.stop, this));
+    if (Notification.permission === 'granted' || Notification.permission === 'denied') { // jshint ignore:line
```

Comment:
you could use `jQuery.inArray()` for that like we do in ruby, no?

```
if(jQuery.inArray(Notification.permission, ['granted', 'denied']) === -1) 
{
  // not found 
} else {
  // found
}
```

Reply:
You made me revisit the documentation. Given that `Notification.permission` [can only have three possible values](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API/Using_the_Notifications_API#checking_current_permission_status), I will change the condition to only check for `default`.

Reply:
I changed the condition. Please take a look again.